File: 231111/연속되는 수 2/continuous-number2.py
N=int(input())

cnt=0
maxnum=0

num=[]

for _ in range(N):
    num.append(int(input()))

for i in range(N):
    if i==0 or num[i]!=num[i-1]:
        cnt=1 
    else:
        cnt+=1
    maxnum=max(cnt,maxnum)

print(maxnum)

File: 251207/연속되는 수 2/continuous-number2.js
const fs = require("fs");
const input = fs.readFileSync(0).toString().trim().split('\n');

const n = Number(input[0]);
const arr = input.slice(1, n + 1).map(Number);

// Please Write your code here.

const cnt=[];
let c=1;
for(let i=0; i<n-1; i++){
    if(arr[i]===arr[i+1]){
        c++;
    }else{
        cnt.push(c);
        c=1;
    }
}

if(n===1){
    console.log(1)
}else{
console.log(Math.max(...cnt))
}
